dynamic_rest/serializers.py

```diff
@@ -76,6 +80,49 @@ def data(self):
                 self._sideloaded_data = ReturnList(data, serializer=self)
         return self._sideloaded_data
 
+    def update(self, queryset, validated_data):
```

Comment:
Can we get this to respect DREST's request style where the data is keyed by the serializer name? e.g these should both work:

```
PATCH /dogs/
{
    "dogs": [{
        "id": 1,
        "fur": "brown"
    }, {
       "id": 2,
       "fur": "tan"
    }]
}
```
and
```
PATCH /dogs/
   [{
        "id": 1,
        "fur": "brown"
    }, {
       "id": 2,
       "fur": "tan"
    }]
```